extensions/http(s)/test/ace/http/request_test.exs

```diff
@@ -49,6 +49,15 @@ defmodule Ace.HTTP.RequestTest do
     assert upload.type == "text/plain"
   end
 
+  test "handles absoluteURI", %{port: port} do
+    request = """
+    GET http://www.raxx.com/ HTTP/1.1
```

Comment:
You are right.`A client MUST send a Host header field in all HTTP/1.1 request
   messages.`